src/library/itunes/itunesmacosimporter.mm

```diff
@@ -89,6 +89,17 @@ int persistentIdToDbId(NSNumber* persistentId) {
         }
     }
 
+    QString uniquifyPlaylistName(QString name) {
+        auto existing = m_playlistDuplicatesByName.find(name);
+        if (existing != m_playlistDuplicatesByName.end()) {
+            m_playlistDuplicatesByName[name] += 1;
+            return QString("%1 #%2").arg(name).arg(m_playlistDuplicatesByName[name]);
```

Comment:
```suggestion
            return name + QString()::number(m_playlistDuplicatesByName[name]);
```
performs faster.

Reply:
I think that's missing the space and the hash, i.e. we would get `Pop2` instead of `Pop #2` for example.

Reply:
I'll close this since the existing iTunes XML importer uses a similar disambiguation mechanism and I assume this is an oversight.